app/plugins/adp/src/components/DeliveryProgramme/CreateDeliveryProgramme.tsx

```diff
@@ -42,6 +46,9 @@ const CreateDeliveryProgramme: React.FC<CreateDeliveryProgrammeProps> = ({
     discoveryApi,
     fetchApi,
   );
+  const { allowed } = usePermission({
```

Comment:
Could this be given a better name?
```suggestion
  const { allowed: allowedToCreateAdpProgramme } = usePermission({
```

Reply:
yep - de-structured 'allowed' for better naming across all